src/js/demo/app.jsx

```diff
@@ -45,6 +45,7 @@ export default class App extends React.Component {
             console.log('Last selected timeslot:');
             console.log(lastSelected);
           } }
+          onMonthChange = { (month) => {console.log(month)}}
```

Comment:
I see no point on doing this (?). If what you're looking to see is whether this works or not, write unit tests instead.